embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/comms/delivery/CacheService.kt

```diff
@@ -87,4 +87,9 @@ internal interface CacheService {
      * @return list of file names
      */
     fun listFilenamesByPrefix(prefix: String): List<String>?
+
+    /**
+     * Loads pending API calls from their old location on disk.
```

Comment:
This method loads the old version of the serialized structure for pending API calls, but the location on disk is the same.